spec/features/cities_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+feature 'City' do
+
+  scenario 'adds a new city' do
+    region = create(:region)
+    expect{
+      create_region(region)
+      create_city
```

Comment:
I have fixed in [09ebdfa](https://github.com/andru1989/population/commit/09ebdfa24687b6ca699e92a4b77610f14cbbf64c) then i'm gonna add the migration to change that restriction in another branch